x-pack/plugins/alerting/server/mocks.ts

```diff
@@ -167,6 +167,9 @@ const createRuleExecutorServicesMock = <
     ruleMonitoringService: createRuleMonitoringServiceMock(),
     share: createShareStartMock(),
     dataViews: dataViewPluginMocks.createStartContract(),
+    indices: {
```

Comment:
This feels like the wrong place for this.  I'm wondering if what's going on is that we want the `getFieldMapping` method of the es client "wrapped" so that we do all the checking with it.  Which would be done here, instead: https://github.com/elastic/kibana/blob/c2e538189e14698080eb1d14b98f6671b3a32186/x-pack/plugins/alerting/server/lib/wrap_scoped_cluster_client.ts#L86-L95

The typing on this stuff is really not great (our fault!), took me a while to chase down some types for this (and now can't find it again!).  I'm wondering if this is really needed, if there is some way to initialize it in the security code instead, since it's not used in the alerting framework.

Reply:
The above comment is a little unclear, so I should note I'm **fine** with wrapping `getFieldMapping` in our wrapped client.  I think that's actually the intention here.  

But if we need to add that `indices` property in the server mocks (as was done in the code that I reviewed), I think we want a different shape for that.

Reply:
Hey @pmuellr,  thanks!
I removed indices mappings (and I put it in wrong place anyway) and everything still works

Reply:
whew!  Not clear if we still have to approve, but ping me if we do.  Or if you have questions about anything.  Otherwise, looks like all SIEM code, so I'll just take a brief scan through ...

Reply:
I don't think I touch any other file which you own. But thank you very much/!